pages/Advanced-Setup.md

```diff
@@ -70,14 +70,15 @@ Example: if a user has a global "fly.use" permission, and then has a negated "fl
 
 * **Temporary permissions will override non-temporary permissions.**
 
-Example: if a user has a false permission set for "test.node", and a temporary true permission set for "test.node", the temporary permission will override the permanent one, and the user will be granted the true node.
+Example: if a user has a false permission set for "test.node", and a temporary true permission set for "test.node", the temporary permission will override the permanent one, and the user will be granted the true node for the duration of it.
```

Comment:
This wording is unclear in that 'it' could mean several things - "for its duration" would clarify that 'it' refers to the true node